src/style.css

```diff
@@ -1 +1,109 @@
-/* Styles go here */
+@font-face {
+  font-family: "Roboto";
+  src: url(/Roboto/Roboto-Medium.ttf);
+  font-style: normal;
+  font-weight: 500;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding-left: 50px;
+  padding-right: 50px;
+  margin: 0;
+  height: 60px;
```

Comment:
Check the check list one more time, please: Don't set height for header explicitly. Let the content (links) dictate it.